src/index.ts

```diff
@@ -20,6 +21,8 @@ router.get("/", async () => {
   });
 });
 
+router.patch("/change-nickname", changeNickname);
```

Comment:
Can we have a better naming of the routes. Routes directly named with functionality is not recommended. Something like `\name\<something>` fits well but feel free to come up with your own.

Reply:
I am trying to keep it in a single name to avoid multiple nesting

Reply:
Everything related to name and namechanges must go under `name` or `<something-else>` path. They will be differentiated based on HTTP methods, sub-paths and path-params

- https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/